content/en/logs/log_collection/_index.md

```diff
@@ -328,6 +328,10 @@ The TCP endpoint is not supported for this site.
 * A log event converted to JSON format should contain less than 256 attributes. Each of those attribute's keys should be less than 50 characters, nested in less than 20 successive levels, and their respective value should be less than 1024 characters if promoted as a facet.
 * Log events can be submitted with a [timestamp][14] that is up to 18h in the past.
 
+<div class="alert alert-info">
+**Preview available**: You can submit logs up to **7 days** old, instead of the current 18h limit. [Register for the Preview](https://www.datadoghq.com/product-preview/ingest-logs-up-to-7-days-old/)
```

Comment:
```suggestion
<b>Preview available</b>: You can submit logs from the past 7 days, instead of the current 18-hour limit. <a href="https://www.datadoghq.com/product-preview/ingest-logs-up-to-7-days-old/">Register for the Preview</a>.
```